frictionless_ckan_mapper/ckan_to_frictionless.py

```diff
@@ -111,6 +111,8 @@ def dataset(ckandict):
     if 'tags' in ckandict:
```

Comment:
the better way to go here would be simply:

```
if ckandict.get('tags'):
    ...
```

Since that will only create keywords if tags in ckandict and it is not empty array.